Cargo.toml

```diff
@@ -36,6 +36,7 @@ log = "0.4.14"
 platforms = { version = "2.0.0", optional = true }
 sass-rs = { version = "0.2.2", optional = true }
 walkdir = { version = "2.3.2", optional = true }
+walrus = { version = "0.19.0", features = ["parallel"] }
```

Comment:
Adding dependencies without using them is not something normal. You need to document a bit.